scripts/example_super_project/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Place this file in a directory with:
+# - farm-ng-core # branch cmake-super-project
```

Comment:
@strasdat this can be done at CI level - handles installing most of the dependencies from source builds.